packages/client-core/src/admin/components/settings/tabs/features.tsx

```diff
@@ -81,12 +81,9 @@ const FeaturesTab = forwardRef(({ open }: { open: boolean }, ref: React.MutableR
       open={open}
     >
       <div className="mt-6 grid grid-cols-1 gap-6">
-        {displayedFeatures.value
-          .toSorted()
-          .sort((x, y) => (x.flagValue === y.flagValue ? 0 : x.flagValue ? -1 : 1)) // show enabled first https://stackoverflow.com/a/17387454/2077741
-          .map((feature) => (
-            <FeatureItem key={feature.id} feature={feature} />
-          ))}
+        {displayedFeatures.value.map((feature) => (
```

Comment:
Maybe sort them in alphabatical order, because otherwise they are displayed in random order.